[36mATRI[0m | 03-07 11:07:12 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7ffa43a5c040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7ffa442e24f0>
         │    └ <function MultiCommand.invoke at 0x7ffa43a600d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7ffa4bbcf370>
           │               │       │       └ <function Command.invoke at 0x7ffa43a5caf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7ffa4bbcf370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7ffa3a9879d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7ffa4bbcf370>
           │   │      │    └ <function run at 0x7ffa3aab28b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7ffa43a588b0>
           └ <click.core.Context object at 0x7ffa4bbcf370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7ffa3aaa2670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7ffa4bc10940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7ffa4bd3e430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7ffa41920790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7ffa4191a310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7ffa3a297940>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7ffa4191a1f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7ffa3a297940>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7ffa4191a280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7ffa3a297940>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7ffa4bc10940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7ffa4bd3e430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-07 11:08:31 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7ffa43a5c040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7ffa442e24f0>
         │    └ <function MultiCommand.invoke at 0x7ffa43a600d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7ffa4bbcf370>
           │               │       │       └ <function Command.invoke at 0x7ffa43a5caf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7ffa4bbcf370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7ffa3a9879d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7ffa4bbcf370>
           │   │      │    └ <function run at 0x7ffa3aab28b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7ffa43a588b0>
           └ <click.core.Context object at 0x7ffa4bbcf370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7ffa3aaa2670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7ffa3cb8b940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7ffa3cb8b4c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7ffa3a8ae670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7ffa3a8aea60>
    └ <uvicorn.server.Server object at 0x7ff9cf0e2d60>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7ffa3a8aeaf0>
           │       │   └ <uvicorn.server.Server object at 0x7ff9cf0e2d60>
           │       └ <function run at 0x7ffa402e5430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7ff9cf0e92c0>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7ffa3cb4c160>: <Task finished name='Task-6541' coro=<_command() done, defined at /usr/local/python3/...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7ff9cc55afa0>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │            │        │    └ GroupMessageEvent(time=1646622505, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7ffa3a9c9670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7ffa3cb4c160>: <Task finished name='Task-6541' coro=<_command() done, defined at /usr/local/python3/...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7ff9cc55afa0>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │       │   │    └ GroupMessageEvent(time=1646622505, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
          │       └ <function Matcher.run at 0x7ffa3cbc8b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7ffa3cb4c160>: <Task finished name='Task-6541' coro=<_command() done, defined at /usr/local/python3/...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7ff9cc55afa0>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │    │          │    └ GroupMessageEvent(time=1646622505, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
          │    └ <function Matcher.simple_run at 0x7ffa3cbc8af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7ffa3038e3a0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>, 'event': GroupMessageEvent(time=1646622505, self_id=5...
                 │    └ <function _ at 0x7ffa3038e3a0>
                 └ <Dependent call=<function _ at 0x7ffa3038e3a0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 458, in _
    await stock.finish(
          │     └ <classmethod object at 0x7ffa3cbc2a90>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 464, in finish
    await cls.send(message, **kwargs)
          │   │    │          └ {'at_sender': True}
          │   │    └ '\n🧾 Successfully sold 1 shares earning 247.0 coins.\nYou have 49 shares left now.\n🟩 Current stock: 247.0 coins,  147.0%'
          │   └ <classmethod object at 0x7ffa3cbc2a30>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 449, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {'at_sender': True}
                 │   │          │              └ '\n🧾 Successfully sold 1 shares earning 247.0 coins.\nYou have 49 shares left now.\n🟩 Current stock: 247.0 coins,  147.0%'
                 │   │          └ GroupMessageEvent(time=1646622505, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │   └ <function Bot.send at 0x7ffa3a9c9b80>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 247, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {'at_sender': True}
                 │    │                      │     │      └ '\n🧾 Successfully sold 1 shares earning 247.0 coins.\nYou have 49 shares left now.\n🟩 Current stock: 247.0 coins,  147.0%'
                 │    │                      │     └ GroupMessageEvent(time=1646622505, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │    │                      └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
                 │    └ <attribute '__class__' of 'object' objects>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 175, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 2560359315, 'group_id': 277620613, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 208, in call_api
    return await super().call_api(api, **data)
                                  │      └ {'user_id': 2560359315, 'group_id': 277620613, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                                  └ 'send_msg'
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 117, in call_api
    raise exception
          └ <ActionFailed data=None, echo={'seq': 3}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 94, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 2560359315, 'group_id': 277620613, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
                   │    │       └ <function Adapter._call_api at 0x7ffa3a9c9f70>
                   │    └ <nonebot.adapters.onebot.v11.adapter.Adapter object at 0x7ffa3aab8880>
                   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/adapter.py", line 107, in _call_api
    return _handle_api_result(
           └ <function _handle_api_result at 0x7ffa3aa5c5e0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/utils.py", line 81, in _handle_api_result
    raise ActionFailed(**result)
          │              └ {'data': None, 'echo': {'seq': 3}, 'msg': 'SEND_MSG_API_ERROR', 'retcode': 100, 'status': 'failed', 'wording': '请参考 go-cqhttp...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: <ActionFailed data=None, echo={'seq': 3}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
[36mATRI[0m | 03-07 11:08:41 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7ffa43a5c040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7ffa442e24f0>
         │    └ <function MultiCommand.invoke at 0x7ffa43a600d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7ffa4bbcf370>
           │               │       │       └ <function Command.invoke at 0x7ffa43a5caf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7ffa4bbcf370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7ffa3a9879d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7ffa4bbcf370>
           │   │      │    └ <function run at 0x7ffa3aab28b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7ffa43a588b0>
           └ <click.core.Context object at 0x7ffa4bbcf370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7ffa3aaa2670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7ffa3cb8b940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7ffa3cb8b4c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7ffa3a8ae670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7ffa3a8aea60>
    └ <uvicorn.server.Server object at 0x7ff9cf0e2d60>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7ffa3a8aeaf0>
           │       │   └ <uvicorn.server.Server object at 0x7ff9cf0e2d60>
           │       └ <function run at 0x7ffa402e5430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7ff9cf0e92c0>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7ffa3cb4c160>: <Task finished name='Task-7396' coro=<_command() done, defined at /usr/local/python3/...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7ff9cc59f550>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │            │        │    └ GroupMessageEvent(time=1646622516, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7ffa3a9c9670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7ffa3cb4c160>: <Task finished name='Task-7396' coro=<_command() done, defined at /usr/local/python3/...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7ff9cc59f550>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │       │   │    └ GroupMessageEvent(time=1646622516, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
          │       └ <function Matcher.run at 0x7ffa3cbc8b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7ffa3cb4c160>: <Task finished name='Task-7396' coro=<_command() done, defined at /usr/local/python3/...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7ff9cc59f550>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell'...
          │    │          │    └ GroupMessageEvent(time=1646622516, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
          │    └ <function Matcher.simple_run at 0x7ffa3cbc8af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7ffa3038e3a0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>, 'event': GroupMessageEvent(time=1646622516, self_id=5...
                 │    └ <function _ at 0x7ffa3038e3a0>
                 └ <Dependent call=<function _ at 0x7ffa3038e3a0>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 458, in _
    await stock.finish(
          │     └ <classmethod object at 0x7ffa3cbc2a90>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 464, in finish
    await cls.send(message, **kwargs)
          │   │    │          └ {'at_sender': True}
          │   │    └ '\n🧾 Successfully sold 1 shares earning 251.0 coins.\nYou have 48 shares left now.\n🟩 Current stock: 251.0 coins,  151.0%'
          │   └ <classmethod object at 0x7ffa3cbc2a30>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 449, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {'at_sender': True}
                 │   │          │              └ '\n🧾 Successfully sold 1 shares earning 251.0 coins.\nYou have 48 shares left now.\n🟩 Current stock: 251.0 coins,  151.0%'
                 │   │          └ GroupMessageEvent(time=1646622516, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │   └ <function Bot.send at 0x7ffa3a9c9b80>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 247, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {'at_sender': True}
                 │    │                      │     │      └ '\n🧾 Successfully sold 1 shares earning 251.0 coins.\nYou have 48 shares left now.\n🟩 Current stock: 251.0 coins,  151.0%'
                 │    │                      │     └ GroupMessageEvent(time=1646622516, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
                 │    │                      └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
                 │    └ <attribute '__class__' of 'object' objects>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 175, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 2560359315, 'group_id': 277620613, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 208, in call_api
    return await super().call_api(api, **data)
                                  │      └ {'user_id': 2560359315, 'group_id': 277620613, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                                  └ 'send_msg'
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 117, in call_api
    raise exception
          └ <ActionFailed data=None, echo={'seq': 5}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 94, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 2560359315, 'group_id': 277620613, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '25...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
                   │    │       └ <function Adapter._call_api at 0x7ffa3a9c9f70>
                   │    └ <nonebot.adapters.onebot.v11.adapter.Adapter object at 0x7ffa3aab8880>
                   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7ff9cde65910>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/adapter.py", line 107, in _call_api
    return _handle_api_result(
           └ <function _handle_api_result at 0x7ffa3aa5c5e0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/utils.py", line 81, in _handle_api_result
    raise ActionFailed(**result)
          │              └ {'data': None, 'echo': {'seq': 5}, 'msg': 'SEND_MSG_API_ERROR', 'retcode': 100, 'status': 'failed', 'wording': '请参考 go-cqhttp...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: <ActionFailed data=None, echo={'seq': 5}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
[36mATRI[0m | 03-07 11:10:17 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7ff7dc406040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7ff7dcc8c4f0>
         │    └ <function MultiCommand.invoke at 0x7ff7dc40a0d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7ff7e4579370>
           │               │       │       └ <function Command.invoke at 0x7ff7dc406af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7ff7e4579370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7ff7d33329d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7ff7e4579370>
           │   │      │    └ <function run at 0x7ff7d345c8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7ff7dc4028b0>
           └ <click.core.Context object at 0x7ff7e4579370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7ff7d344b670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7ff7e45ba940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7ff7e46e8430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7ff7da2ca790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7ff7da2c4310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7ff7d2c3eb20>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7ff7da2c41f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7ff7d2c3eb20>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7ff7da2c4280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7ff7d2c3eb20>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7ff7e45ba940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7ff7e46e8430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-07 11:12:26 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f2464064040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f24648ea4f0>
         │    └ <function MultiCommand.invoke at 0x7f24640680d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f246c1d7370>
           │               │       │       └ <function Command.invoke at 0x7f2464064af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f246c1d7370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f245af8f9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f246c1d7370>
           │   │      │    └ <function run at 0x7f245b0ba8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f24640608b0>
           └ <click.core.Context object at 0x7f246c1d7370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f245b0a9670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f246c218940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f246c346430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f2461f28790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f2461f22310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f245a89e910>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f2461f221f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f245a89e910>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7f2461f22280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f245a89e910>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f246c218940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7f246c346430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-07 11:16:10 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f0dcba1f040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f0dcc2a54f0>
         │    └ <function MultiCommand.invoke at 0x7f0dcba230d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f0dd3b92370>
           │               │       │       └ <function Command.invoke at 0x7f0dcba1faf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f0dd3b92370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f0dc294a9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f0dd3b92370>
           │   │      │    └ <function run at 0x7f0dc2a758b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f0dcba1b8b0>
           └ <click.core.Context object at 0x7f0dd3b92370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f0dc2a64670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f0dd3bd3940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f0dd3d01430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f0dc98e3790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f0dc98dd310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f0dc22599a0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f0dc98dd1f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f0dc22599a0>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7f0dc98dd280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f0dc22599a0>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f0dd3bd3940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7f0dd3d01430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-07 11:16:20 | ERROR plugin.manager >> Failed to import "nonebot_plugin_russian"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f0dcba1f040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f0dcc2a54f0>
         │    └ <function MultiCommand.invoke at 0x7f0dcba230d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f0dd3b92370>
           │               │       │       └ <function Command.invoke at 0x7f0dcba1faf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f0dd3b92370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f0dc294a9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f0dd3b92370>
           │   │      │    └ <function run at 0x7f0dc2a758b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f0dcba1b8b0>
           └ <click.core.Context object at 0x7f0dd3b92370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f0dc2a64670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f0dd3bd3940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f0dd3d01430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f0dc98e3790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f0dc98dd310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f0dc22599a0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f0dc98dd1f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f0dc22599a0>
                  │    │           │         └ 'nonebot_plugin_russian'
                  │    │           └ 'nonebot_plugin_russian'
                  │    └ <function PluginManager.load_plugin at 0x7f0dc98dd280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f0dc22599a0>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f0dd3bd3940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_russian'
           │          └ <function _gcd_import at 0x7f0dd3d01430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_russian' from './src/plugins/nonebot_plugin_russian/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 15, in <module>
    from .data_source import russian_manager, max_bet_gold

  File "./src/plugins/nonebot_plugin_russian/data_source.py", line 668
    update_price = float(self._check_stock_handle()["money_per_stock"])+delta)
                                                                             ^

SyntaxError: unmatched ')'
[36mATRI[0m | 03-07 11:18:45 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f7f0e839040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f7f0f0bf4f0>
         │    └ <function MultiCommand.invoke at 0x7f7f0e83d0d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f7f169ac370>
           │               │       │       └ <function Command.invoke at 0x7f7f0e839af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f7f169ac370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f7f057649d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f7f169ac370>
           │   │      │    └ <function run at 0x7f7f0588f8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f7f0e8358b0>
           └ <click.core.Context object at 0x7f7f169ac370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f7f0587e670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f7f169ed940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f7f16b1b430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f7f0c6fd790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f7f0c6f7310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f7f05073940>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f7f0c6f71f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f7f05073940>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7f7f0c6f7280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f7f05073940>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f7f169ed940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7f7f16b1b430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-07 11:48:27 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f3fb9eff040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f3fba7854f0>
         │    └ <function MultiCommand.invoke at 0x7f3fb9f030d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f3fc2072370>
           │               │       │       └ <function Command.invoke at 0x7f3fb9effaf0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f3fc2072370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f3fb0e2b9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f3fc2072370>
           │   │      │    └ <function run at 0x7f3fb0f558b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f3fb9efb8b0>
           └ <click.core.Context object at 0x7f3fc2072370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f3fb0f44670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f3fc20b2940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f3fc21e1430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f3fb7dc3790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f3fb7dbd310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f3fb0737a60>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f3fb7dbd1f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f3fb0737a60>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7f3fb7dbd280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f3fb0737a60>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f3fc20b2940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7f3fc21e1430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-07 11:49:22 | ERROR plugin.manager >> Failed to import "nonebot_plugin_statistical"
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f2326924040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f23271aa4f0>
         │    └ <function MultiCommand.invoke at 0x7f23269280d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f232ea97370>
           │               │       │       └ <function Command.invoke at 0x7f2326924af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f232ea97370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f231d84f9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f232ea97370>
           │   │      │    └ <function run at 0x7f231d97a8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f23269208b0>
           └ <click.core.Context object at 0x7f232ea97370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f231d969670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 19, in run_bot
    module = importlib.import_module(module_name)
             │         │             └ 'bot'
             │         └ <function import_module at 0x7f232ead7940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'bot'
           │          └ <function _gcd_import at 0x7f232ec06430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./bot.py", line 29, in <module>
    nonebot.load_plugins("src/plugins")
    │       └ <function load_plugins at 0x7f23247e8790>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/load.py", line 39, in load_plugins
    return manager.load_all_plugins()
           │       └ <function PluginManager.load_all_plugins at 0x7f23247e2310>
           └ <nonebot.plugin.manager.PluginManager object at 0x7f231d15ca60>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 128, in load_all_plugins
    return set(
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 129, in <genexpr>
    filter(None, (self.load_plugin(name) for name in self.list_plugins()))
                  │    │           │         │       │    └ <function PluginManager.list_plugins at 0x7f23247e21f0>
                  │    │           │         │       └ <nonebot.plugin.manager.PluginManager object at 0x7f231d15ca60>
                  │    │           │         └ 'nonebot_plugin_statistical'
                  │    │           └ 'nonebot_plugin_statistical'
                  │    └ <function PluginManager.load_plugin at 0x7f23247e2280>
                  └ <nonebot.plugin.manager.PluginManager object at 0x7f231d15ca60>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 106, in load_plugin
    module = importlib.import_module(
             │         └ <function import_module at 0x7f232ead7940>
             └ <module 'importlib' from '/usr/local/python3/lib/python3.8/importlib/__init__.py'>
  File "/usr/local/python3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           │          │           │    │        │        └ 0
           │          │           │    │        └ None
           │          │           │    └ 0
           │          │           └ 'src.plugins.nonebot_plugin_statistical'
           │          └ <function _gcd_import at 0x7f232ec06430>
           └ <module 'importlib._bootstrap' (frozen)>
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/plugin/manager.py", line 194, in exec_module
    super().exec_module(module)
                        └ <module 'src.plugins.nonebot_plugin_statistical' from './src/plugins/nonebot_plugin_statistical/__init__.py'>
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed

  File "./src/plugins/nonebot_plugin_statistical/__init__.py", line 1, in <module>
    from .statistical_hook import *

  File "./src/plugins/nonebot_plugin_statistical/statistical_hook.py", line 7, in <module>
    from .config import save_data, get_prefix_group_count_dict, get_prefix_user_count_dict, \

  File "./src/plugins/nonebot_plugin_statistical/config.py", line 149, in <module>
    plugin2cmd: dict = json.load(f)
                       │    │    └ <_io.TextIOWrapper name='/root/Projects/Kori-Bot/Kori-bot/data/statistical/plugin2cmd.json' mode='r' encoding='utf8'>
                       │    └ <built-in function load>
                       └ <module 'ujson' from '/usr/local/python3/lib/python3.8/site-packages/ujson.cpython-38-x86_64-linux-gnu.so'>

ValueError: Expected object or value
[36mATRI[0m | 03-07 14:40:39 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f2326924040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f23271aa4f0>
         │    └ <function MultiCommand.invoke at 0x7f23269280d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f232ea97370>
           │               │       │       └ <function Command.invoke at 0x7f2326924af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f232ea97370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f231d84f9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f232ea97370>
           │   │      │    └ <function run at 0x7f231d97a8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f23269208b0>
           └ <click.core.Context object at 0x7f232ea97370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f231d969670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7f231fa53940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7f231fa534c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7f231d777670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7f231d777a60>
    └ <uvicorn.server.Server object at 0x7f22b24c2430>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7f231d777af0>
           │       │   └ <uvicorn.server.Server object at 0x7f22b24c2430>
           │       └ <function run at 0x7f23231ac430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7f22b24d9340>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-430741' coro=<_command() done, defined at /usr/local/python...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7f22b09c4670>
          │            │        │    │      └ {'_prefix': {'command': ('lottery',), 'raw_command': '/lottery', 'command_arg': []}}
          │            │        │    └ GroupMessageEvent(time=1646635233, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7f231d891670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-430741' coro=<_command() done, defined at /usr/local/python...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7f22b09c4670>
          │       │   │    │      └ {'_prefix': {'command': ('lottery',), 'raw_command': '/lottery', 'command_arg': []}}
          │       │   │    └ GroupMessageEvent(time=1646635233, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │       └ <function Matcher.run at 0x7f231fa90b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-430741' coro=<_command() done, defined at /usr/local/python...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7f22b09c4670>
          │    │          │    │      └ {'_prefix': {'command': ('lottery',), 'raw_command': '/lottery', 'command_arg': []}}
          │    │          │    └ GroupMessageEvent(time=1646635233, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │    └ <function Matcher.simple_run at 0x7f231fa90af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7f2303c79430>, params=[ModelField(name='event', type=GroupMessageEvent, required=True), Mode...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'event': GroupMessageEvent(time=1646635233, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, m...
                 │    └ <function _ at 0x7f2303c79430>
                 └ <Dependent call=<function _ at 0x7f2303c79430>, params=[ModelField(name='event', type=GroupMessageEvent, required=True), Mode...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 336, in _
    await lottery.send(message=msg, at_sender=True)
          │       │            └ '\n💸 Successfully bought 1 lottery ticket using 50 coins.\n🎉 YOU WON!!!!! 855.0 coins had been transferred to your pocket.'
          │       └ <classmethod object at 0x7f231fa87a30>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>

  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 449, in send
    return await bot.send(event=event, message=_message, **kwargs)
                 │   │          │              │           └ {'at_sender': True}
                 │   │          │              └ '\n💸 Successfully bought 1 lottery ticket using 50 coins.\n🎉 YOU WON!!!!! 855.0 coins had been transferred to your pocket.'
                 │   │          └ GroupMessageEvent(time=1646635233, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
                 │   └ <function Bot.send at 0x7f231d891b80>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 247, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {'at_sender': True}
                 │    │                      │     │      └ '\n💸 Successfully bought 1 lottery ticket using 50 coins.\n🎉 YOU WON!!!!! 855.0 coins had been transferred to your pocket.'
                 │    │                      │     └ GroupMessageEvent(time=1646635233, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
                 │    │                      └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
                 │    └ <attribute '__class__' of 'object' objects>
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 175, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 1834580024, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '18...
                 └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/bot.py", line 208, in call_api
    return await super().call_api(api, **data)
                                  │      └ {'user_id': 1834580024, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '18...
                                  └ 'send_msg'
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 117, in call_api
    raise exception
          └ <ActionFailed data=None, echo={'seq': 91}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/adapter/bot.py", line 94, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 1834580024, 'group_id': 937760225, 'message_type': 'group', 'message': [MessageSegment(type='at', data={'qq': '18...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
                   │    │       └ <function Adapter._call_api at 0x7f231d891f70>
                   │    └ <nonebot.adapters.onebot.v11.adapter.Adapter object at 0x7f231d97f9a0>
                   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/adapter.py", line 107, in _call_api
    return _handle_api_result(
           └ <function _handle_api_result at 0x7f231d9235e0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/adapters/onebot/v11/utils.py", line 81, in _handle_api_result
    raise ActionFailed(**result)
          │              └ {'data': None, 'echo': {'seq': 91}, 'msg': 'SEND_MSG_API_ERROR', 'retcode': 100, 'status': 'failed', 'wording': '请参考 go-cqhtt...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: <ActionFailed data=None, echo={'seq': 91}, msg=SEND_MSG_API_ERROR, retcode=100, status=failed, wording=请参考 go-cqhttp 端输出>
[36mATRI[0m | 03-07 21:41:51 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f2326924040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f23271aa4f0>
         │    └ <function MultiCommand.invoke at 0x7f23269280d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f232ea97370>
           │               │       │       └ <function Command.invoke at 0x7f2326924af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f232ea97370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f231d84f9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f232ea97370>
           │   │      │    └ <function run at 0x7f231d97a8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f23269208b0>
           └ <click.core.Context object at 0x7f232ea97370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f231d969670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7f231fa53940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7f231fa534c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7f231d777670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7f231d777a60>
    └ <uvicorn.server.Server object at 0x7f22b24c2430>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7f231d777af0>
           │       │   └ <uvicorn.server.Server object at 0x7f22b24c2430>
           │       └ <function run at 0x7f23231ac430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7f22b24d9340>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-1438388' coro=<_command() done, defined at /usr/local/pytho...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7f22872566d0>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │            │        │    └ GroupMessageEvent(time=1646660511, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7f231d891670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-1438388' coro=<_command() done, defined at /usr/local/pytho...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7f22872566d0>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │       │   │    └ GroupMessageEvent(time=1646660511, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │       └ <function Matcher.run at 0x7f231fa90b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-1438388' coro=<_command() done, defined at /usr/local/pytho...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7f22872566d0>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │    │          │    └ GroupMessageEvent(time=1646660511, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │    └ <function Matcher.simple_run at 0x7f231fa90af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7f2303c79820>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>, 'event': GroupMessageEvent(time=1646660511, self_id=5...
                 │    └ <function _ at 0x7f2303c79820>
                 └ <Dependent call=<function _ at 0x7f2303c79820>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 455, in _
    number = int(number)
                 └ '0.1'

ValueError: invalid literal for int() with base 10: '0.1'
[36mATRI[0m | 03-07 21:42:05 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f2326924040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f23271aa4f0>
         │    └ <function MultiCommand.invoke at 0x7f23269280d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f232ea97370>
           │               │       │       └ <function Command.invoke at 0x7f2326924af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f232ea97370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f231d84f9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f232ea97370>
           │   │      │    └ <function run at 0x7f231d97a8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f23269208b0>
           └ <click.core.Context object at 0x7f232ea97370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f231d969670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7f231fa53940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7f231fa534c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7f231d777670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7f231d777a60>
    └ <uvicorn.server.Server object at 0x7f22b24c2430>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7f231d777af0>
           │       │   └ <uvicorn.server.Server object at 0x7f22b24c2430>
           │       └ <function run at 0x7f23231ac430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7f22b24d9340>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-1439183' coro=<_command() done, defined at /usr/local/pytho...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7f2287256d00>
          │            │        │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │            │        │    └ GroupMessageEvent(time=1646660525, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7f231d891670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-1439183' coro=<_command() done, defined at /usr/local/pytho...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7f2287256d00>
          │       │   │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │       │   │    └ GroupMessageEvent(time=1646660525, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │       └ <function Matcher.run at 0x7f231fa90b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-1439183' coro=<_command() done, defined at /usr/local/pytho...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7f2287256d00>
          │    │          │    │      └ {'_prefix': {'command': ('stock',), 'raw_command': '/stock', 'command_arg': [MessageSegment(type='text', data={'text': 'sell ...
          │    │          │    └ GroupMessageEvent(time=1646660525, self_id=507346814, post_type='message', sub_type='normal', user_id=2560359315, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │    └ <function Matcher.simple_run at 0x7f231fa90af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7f2303c79820>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'bot': <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>, 'event': GroupMessageEvent(time=1646660525, self_id=5...
                 │    └ <function _ at 0x7f2303c79820>
                 └ <Dependent call=<function _ at 0x7f2303c79820>, params=[ModelField(name='bot', type=Bot, required=True), ModelField(name='eve...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 455, in _
    number = int(number)
                 └ '0.1'

ValueError: invalid literal for int() with base 10: '0.1'
[36mATRI[0m | 03-08 08:43:15 | ERROR nonebot >> Running matcher <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False> failed.
Traceback (most recent call last):

  File "/usr/bin/nb", line 8, in <module>
    sys.exit(main())
    │   │    └ <ClickAliasedGroup main>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7f2326924040>
           └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7f23271aa4f0>
         │    └ <function MultiCommand.invoke at 0x7f23269280d0>
         └ <ClickAliasedGroup main>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           │               │       │       │      └ <click.core.Context object at 0x7f232ea97370>
           │               │       │       └ <function Command.invoke at 0x7f2326924af0>
           │               │       └ <ClickAliasedCommand run>
           │               └ <click.core.Context object at 0x7f232ea97370>
           └ <function MultiCommand.invoke.<locals>._process_result at 0x7f231d84f9d0>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'file': 'bot.py', 'app': 'app'}
           │   │      │    │           └ <click.core.Context object at 0x7f232ea97370>
           │   │      │    └ <function run at 0x7f231d97a8b0>
           │   │      └ <ClickAliasedCommand run>
           │   └ <function Context.invoke at 0x7f23269208b0>
           └ <click.core.Context object at 0x7f232ea97370>
  File "/usr/local/python3/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
                       │       └ {'file': 'bot.py', 'app': 'app'}
                       └ ()
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/commands/main.py", line 30, in run
    run_bot(file, app)
    │       │     └ 'app'
    │       └ 'bot.py'
    └ <function run_bot at 0x7f231d969670>
  File "/usr/local/python3/lib/python3.8/site-packages/nb_cli/handlers/deploy.py", line 24, in run_bot
    nonebot.run(app=f"{module_name}:{app}")
    │       └ <function run at 0x7f231fa53940>
    └ <module 'nonebot' from '/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/__init__.py", line 258, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {'app': 'bot:app'}
    │                 └ ()
    └ <function get_driver at 0x7f231fa534c0>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/drivers/fastapi.py", line 168, in run
    uvicorn.run(
    │       └ <function run at 0x7f231d777670>
    └ <module 'uvicorn' from '/usr/local/python3/lib/python3.8/site-packages/uvicorn/__init__.py'>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/main.py", line 463, in run
    server.run()
    │      └ <function Server.run at 0x7f231d777a60>
    └ <uvicorn.server.Server object at 0x7f22b24c2430>
  File "/usr/local/python3/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x7f231d777af0>
           │       │   └ <uvicorn.server.Server object at 0x7f22b24c2430>
           │       └ <function run at 0x7f23231ac430>
           └ <module 'asyncio' from '/usr/local/python3/lib/python3.8/asyncio/__init__.py'>
  File "/usr/local/python3/lib/python3.8/asyncio/runners.py", line 43, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x7f22b24d9340>
           │    └ <method 'run_until_complete' of 'uvloop.loop.Loop' objects>
           └ <uvloop.Loop running=True closed=False debug=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 153, in _check_matcher
    await _run_matcher(Matcher, bot, event, state, stack, dependency_cache)
          │            │        │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-2705008' coro=<_command() done, defined at /usr/local/pytho...
          │            │        │    │      │      └ <contextlib.AsyncExitStack object at 0x7f2287236e20>
          │            │        │    │      └ {'_prefix': {'command': ('lottery',), 'raw_command': '/lottery', 'command_arg': [MessageSegment(type='text', data={'text': 'b...
          │            │        │    └ GroupMessageEvent(time=1646700195, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
          │            │        └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │            └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
          └ <function _run_matcher at 0x7f231d891670>
> File "/usr/local/python3/lib/python3.8/site-packages/nonebot/message.py", line 202, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-2705008' coro=<_command() done, defined at /usr/local/pytho...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x7f2287236e20>
          │       │   │    │      └ {'_prefix': {'command': ('lottery',), 'raw_command': '/lottery', 'command_arg': [MessageSegment(type='text', data={'text': 'b...
          │       │   │    └ GroupMessageEvent(time=1646700195, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
          │       │   └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │       └ <function Matcher.run at 0x7f231fa90b80>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 668, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _command at 0x7f231fa14160>: <Task finished name='Task-2705008' coro=<_command() done, defined at /usr/local/pytho...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x7f2287236e20>
          │    │          │    │      └ {'_prefix': {'command': ('lottery',), 'raw_command': '/lottery', 'command_arg': [MessageSegment(type='text', data={'text': 'b...
          │    │          │    └ GroupMessageEvent(time=1646700195, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, message_typ...
          │    │          └ <nonebot.adapters.onebot.v11.bot.Bot object at 0x7f22b1234670>
          │    └ <function Matcher.simple_run at 0x7f231fa90af0>
          └ <Matcher from src.plugins.nonebot_plugin_russian, type=message, priority=5, temp=False>
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/internal/matcher.py", line 635, in simple_run
    await handler(
          └ <Dependent call=<function _ at 0x7f2303c79430>, params=[ModelField(name='event', type=GroupMessageEvent, required=True), Mode...
  File "/usr/local/python3/lib/python3.8/site-packages/nonebot/dependencies/__init__.py", line 92, in __call__
    return await self.call(**values)
                 │    │      └ {'event': GroupMessageEvent(time=1646700195, self_id=507346814, post_type='message', sub_type='normal', user_id=1834580024, m...
                 │    └ <function _ at 0x7f2303c79430>
                 └ <Dependent call=<function _ at 0x7f2303c79430>, params=[ModelField(name='event', type=GroupMessageEvent, required=True), Mode...

  File "./src/plugins/nonebot_plugin_russian/__init__.py", line 309, in _
    arg = int(arg)
              └ 'buy 1'

ValueError: invalid literal for int() with base 10: 'buy 1'
